clio-server/src/main/scala/org/broadinstitute/clio/server/service/PersistenceService.scala

```diff
@@ -33,13 +33,14 @@ class PersistenceService private (persistenceDAO: PersistenceDAO, searchDAO: Sea
   def upsertMetadata[TK, TM, D <: ClioDocument: ElasticsearchIndex](
     transferKey: TK,
     transferMetadata: TM,
-    documentMapper: ElasticsearchDocumentMapper[TK, TM, D]
+    documentMapper: ElasticsearchDocumentMapper[TK, TM, D],
+    elasticsearchIndex: ElasticsearchIndex[D]
```

Comment:
Does it work if `elasticsearchIndex` is `implicit` too? Then we won't have to pass things explicitly below.